crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -39,24 +39,29 @@ impl<'a, W> TerminalPane<'a, W> {
     }
 
     fn footer(&self) -> Line {
-        let task_list_message = if !self.has_sidebar {
-            TASK_LIST_HIDDEN
-        } else {
-            ""
+        let build_message_vec = |footer_text: &str| -> String {
+            let mut messages = vec![footer_text];
+
+            if !self.has_sidebar {
+                messages.push(TASK_LIST_HIDDEN);
+            }
+
+            if self.terminal_output.has_selection() {
+                messages.push(HAS_SELECTION);
+            }
+
+            // Spaces are used to pad the footer text for aesthetics
+            ["   ".into(), messages.join(", ")].join("")
```

Comment:
I think directly formatting this instead of `.join("")`ing an array is cleaner. Also one less allocation (not that it matters)
```suggestion
            format!("   {}", messages.join(", "))
```

Reply:
And you're right. These are the dumb things I write after midnight.